javascript/node/selenium-webdriver/test/lib/capabilities_test.js

```diff
@@ -126,4 +127,21 @@ describe('Capabilities', function() {
       assert.deepEqual({bar: 123}, caps[Symbols.serialize]());
     });
   });
+
+  describe('Test StrictFileInteractability capability', function(env){
```

Comment:
Add a test for a false capability as well

Reply:
Added test with capability setStrictFileInteractability(false)